ansible/playbooks/AdoptOpenJDK_ITW_Playbook/roles/Common/scripts/install-xcode.sh

```diff
@@ -20,25 +24,34 @@ fi
 # instead from public download URLs, which can be found in the dvtdownloadableindex:
 # https://devimages.apple.com.edgekey.net/downloads/xcode/simulators/index-3905972D-B609-49CE-8D06-51ADC78E07BC.dvtdownloadableindex
 
-if [[ "$osx_vers" -eq 7 ]] || [[ "$osx_vers" -eq 8 ]]; then
+if [[ "$os_maj_vers" -eq 10 ]] && [[ "$osx_vers" -eq 7 ]] || [[ "$osx_vers" -eq 8 ]]; then
```

Comment:
How does bash handle  && || precendence?
Does this need to be
```
[[ "$os_maj_vers" -eq 10 ]] && ([[ "$osx_vers" -eq 7 ]] || [[ "$osx_vers" -eq 8 ]])
```

Reply:
It does indeed need amending to be explicit... now done.